src/services/SyncService.js

```diff
@@ -95,12 +95,10 @@ class SyncService {
 			document: this.document,
 			session: this.session,
 		})
-		const content = connectionData.content
-			|| await this._fetchDocument()
 		this.emit('loaded', {
 			document: this.document,
 			session: this.session,
-			documentSource: '' + content,
+			documentSource: '' + connectionData.content,
```

Comment:
Isn't this the same thing we removed above? I'm confused. :sweat_smile:

Reply:
Yes, which is fine still just without the fetch fallback

Reply:
The main problem was that an empty string (from a file without content) would always trigger the fetch request which is actually not necessary anymore as we always pass the content earlier.